src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Here, the use of iteration methods such as `split`, `map`, and `filter` is appropriate and aligns with good coding practice.